homeassistant/brands/ikea.json

```diff
@@ -0,0 +1,6 @@
+{
+  "domain": "tradfri",
```

Comment:
Should this be something like `ikea`? It may not matter functionally, but it could help future devs.

Reply:
Yes, it should be but this seemed to hint that the domain should be tradfri:
https://github.com/home-assistant/brands/pull/1065

Reply:
Got it. That change was before brands existed, so I think the better long-term change is to update the brands repo to contain `ikea` assets that are linked to `tradfri` (as an example, see what I'm now doing with AirVisual as a master brand: https://github.com/home-assistant/brands/pull/3750).

Reply:
OK, creating a brands PR. If I am renaming a domain, is there anywhere else it needs to be changed? In the tradfri integration itself?

Reply:
You aren't renaming the domain used in code – this is a brand domain. So, in the brands repo, you'll establish that under `core_brands`, then symlink that to `tradfri` (similar to what I shared with AirVisual).

Reply:
Hopefully I have done that now in https://github.com/home-assistant/brands/pull/3752. Althogh I've not worked with symlinks in git before and have been unable to test on my machine

Reply:
Sounds good. FYI, I'm fairly new to brands myself, so to make sure we're looking at this correctly, going to mark this for a second opinion.